gke-deploy/doc/gke-deploy_apply.md

```diff
@@ -34,12 +34,12 @@ gke-deploy apply [flags]
 
 ```
   -c, --cluster string     Name of GKE cluster to deploy to.
-  -f, --filename string    Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with ".yml" or ".yaml").
+  -f, --filename string    Configuration file or directory of configuration files to use to create Kubernetes objects (file or files in directory must end with ".yml" or ".yaml"). The file or directory could also be a GCS url.
```

Comment:
Could you generate the docs by running go run doc-gen/main.go --dir=doc

Reply:
Done.